MIDIchlorians/MIDIchlorians/AnimationSequence.swift

```diff
@@ -13,12 +13,20 @@ class AnimationSequence {
     private var tickCounter: Int
     var toBeRemoved: Bool
     var name: String?
+    private var frequencyPerBeat: BeatFrequency = BeatFrequency.eight
```

Comment:
Made `BeatFrequency.eight` as default since that corresponds to the frame rate before.